config/environments/production.rb

```diff
@@ -76,4 +76,14 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+  port: "587",
+  address: "smtp.mandrillapp.com",
+  user_name: ENV["MANDRILL_USERNAME"],
+  password: ENV["MANDRILL_APIKEY"],
+  domain: "heroku.com",
+  authentication: :plain
+}
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.